daphne_server/src/roles/aggregator.rs

```diff
@@ -66,21 +144,97 @@ impl<S: Sync> DapAggregator<S> for crate::App {
     async fn unauthorized_reason(
```

Comment:
The implementation appears to be line-for-line the same as for daphne_worker. As we discussed previously, the TLS certificate bits are specific to Workers. In Oxy this is going to work differently, and I don't think we want to force ourselves to match Workers here. For example, `verified == "SUCCESS"` is a super awkward way to check a valid certificate.

I'd suggest that we put a pin in this. For now I would replace the TLS cert code path here with a `todo!()` so that we only support bearer tokens for now.

In any case, this code is very delicate. I think we need to refactor so that all of the code shared by daphne_worker and daphne_server is implemented by the same function.